data-otservbr-global/scripts/creaturescripts/others/player_death.lua

```diff
@@ -70,14 +70,8 @@ function playerDeath.onDeath(player, corpse, killer, mostDamageKiller, unjustifi
 	-- Start Webhook Player Death
 	local playerName = player:getName()
```

Comment:
we can remove this because it isn't in use. We are using in `getMarkdownLink` function on data/events/scripts/player.lua